lesson_08/cpp/cyclic_functions.cpp

```diff
@@ -0,0 +1,69 @@
+#include "../headers/cyclic_functions.hpp"
+
+void count_numbers_by_sign()
+{
+	int numbers_qty{ 0 }, positive_qty{ 0 }, negative_qty{ 0 }, zero_qty{ 0 };
+
+	std::cout << "Enter numbers quantity:\n";
+	std::cin >> numbers_qty;
+
+	while (--numbers_qty >= 0)
```

Comment:
Prefer `for` loop for simple iteration with index

Reply:
Fixed. In this case with "while()" I do not have useless "i" variable.